.github/workflows/copilot-setup-steps.yml

```
@@ -0,0 +1,64 @@
+name: Copilot Setup Steps
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  copilot-setup-steps:
+    name: Pre-install Dependencies for Copilot Environment
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Setup .NET 6.0.x
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 6.0.x
```

```suggestion
        dotnet-version: 6.0.0
```

I think we might literally need dotnet version `6.0.0` for the build to work, but I could be wrong.